go.mod

```diff
@@ -10,8 +10,7 @@ require (
 	github.com/DataDog/datadog-go v3.2.0+incompatible
 	github.com/DataDog/zstd v1.4.0
 	github.com/Masterminds/squirrel v0.0.0-20190107164353-fa735ea14f09
-	github.com/Microsoft/go-winio v0.4.14 // indirect
```

Comment:
Hey @cirocosta 
Looks like we dropped `go-winio` in go.mod, however, in `go.sum` it just appears as though the sha changed. 
Are we missing an update in `go.sum`, where `go-winio` is removed ?

Reply:
Hey @xtreme-sameer-vohra , I don't think an update is missing as the commit is literally the result of running `go mod tidy`, which is what can give us the full updated list:

```
go mod help tidy
usage: go mod tidy [-v]

Tidy makes sure go.mod matches the source code in the module.
It adds any missing modules necessary to build the current module's
packages and dependencies, and it removes unused modules that
don't provide any relevant packages. It also adds any missing entries
to go.sum and removes any unnecessary ones.
```

now, *why does this show in `go.sum` but not `go.mod`?* I'm not entirely sure.

My guess is that as this comes from `github.com/containerd/containerd`

```
go mod why github.com/Microsoft/go-winio
# github.com/Microsoft/go-winio
github.com/concourse/concourse/worker/backend
github.com/containerd/containerd/cio
github.com/Microsoft/go-winio
```

which is *not* module aware (it uses a `vendor` directory), Go is not keeping track of it as a module dependency, but because `go-winio` itself is a module, which has a SHA associated with it, it lets us verify that we're getting what we expect, even though its direct dependent (`containerd`) is not module aware.